examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
something is weird here. Take a look:
![weird](https://cloud.githubusercontent.com/assets/15852194/12061610/ba7f156e-afb0-11e5-9633-d7ae8ca22efd.png)

why does it look so different from SSP + notch? all the peaks are pointing downwards here but in SSP + Notch, they aren't.